src/main/java/godaddy/registry/jtoolkit2/se/fee/DomainApplicationFeeResponseExtension.java

```diff
@@ -55,16 +55,17 @@ public void fromXML(XMLDocument xmlDoc) throws XPathExpressionException {
             int nodeLength = feeNodeList.getLength();
             for (int i = 0; i < nodeLength; i++) {
                 Node node = feeNodeList.item(i);
-                String description = node.getAttributes().getNamedItem("description").getNodeValue();
-                if (description.equals("Application Fee")) {
-                    applicationFee = new BigDecimal(node.getTextContent());
+                if (node.getAttributes().getNamedItem("description") != null) {
```

Comment:
Similar to the other comments - could you please extract a variable for `node.getAttributes().getNamedItem("description")` so that we don't have to repeat on line 58 and 59?

Reply:
DOne